src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php

```diff
@@ -64,7 +65,14 @@ public function setContainer(ContainerInterface $container)
     protected function getParameter(string $name)
     {
         if (!$this->container->has('parameter_bag')) {
-            throw new \LogicException('The "parameter_bag" service is not available. Try running "composer require dependency-injection:^4.1"');
+            $message = 'The "parameter_bag" service could not be located. ';
+            if (!interface_exists(ContainerBagInterface::class)) {
+                $message .= 'Upgrade symfony/dependency-injection to at least 4.1.0 to have it injected automatically.';
```

Comment:
suggesting to install/upgrade symfony/dependency-injection does not seem relevant as framework-bundle requires it `^4.1`

Reply:
Lol, hadn't bothered to doublecheck that because it was the original message and I assumed it to have a reason.